extensions/todoist/src/components/TaskActions.tsx

```diff
@@ -1,4 +1,14 @@
-import { Action, ActionPanel, Color, Icon, Toast, confirmAlert, showToast, useNavigation } from "@raycast/api";
+import {
+  Action,
+  ActionPanel,
+  Color,
+  Icon,
```

Comment:
```suggestion
  Icon,
  Keyboard,
```

Just to fix the import issue in case you apply suggestions directly from PR.